kong/status/ready.lua

```diff
@@ -0,0 +1,111 @@
+local declarative = require "kong.db.declarative"
+local constants = require "kong.constants"
+
+local tonumber = tonumber
+local kong = kong
+
+local get_current_hash = declarative.get_current_hash
+
+
+local worker_count = ngx.worker.count()
+local kong_shm     = ngx.shared.kong
+
+local ngx_log        = ngx.log
+local ngx_WARN       = ngx.WARN
+
+local is_dbless = kong.configuration.database == "off"
+local is_control_plane = kong.configuration.role == "control_plane"
+
+local DECLARATIVE_PLUGINS_REBUILD_COUNT_KEY = 
+                                constants.DECLARATIVE_PLUGINS_REBUILD_COUNT_KEY
+local DECLARATIVE_ROUTERS_REBUILD_COUNT_KEY =
+                                constants.DECLARATIVE_ROUTERS_REBUILD_COUNT_KEY
+local DECLARATIVE_EMPTY_CONFIG_HASH = constants.DECLARATIVE_EMPTY_CONFIG_HASH
+
+
+local function is_dbless_ready(router_rebuilds, plugins_iterator_rebuilds)
+  if router_rebuilds < worker_count then
+    return false, "router rebuilds are not complete"
```

Comment:
Please change this error message to something like: "router builds not yet complete, router ready on 3 of 4 workers".
In this example, 4 is the total number of workers and 3 is the worker count on which router has been built

Reply:
OK